tests/validation_tests/test_failures.py

```diff
@@ -255,6 +269,18 @@ def test_invalid_enum_messages():
         'For column "Proximity", you have entered "Value" which isn\'t correct. You '
         "must select an option from the list provided",
     )
+    assert failure13.to_message() == (
```

Comment:
I think this assertion naming clashes with the previous PR I just approved, which also has a 13